Anastasia Dubilo/webpack.config.js

```diff
@@ -2,18 +2,22 @@ const path = require('path');
 
 module.exports = {
     entry: './src/App.js',
-    output: {
-        path: path.resolve(__dirname, 'dist'),
-        filename: 'main.js'
+    output:{
+        filename: 'main.js',
+        path: path.resolve(__dirname, 'dist')
     },
-    mode: 'development',
-    module: {
+    module:{
         rules: [
             {
-                test: /\.js$/,
+                test: /\.js?$/,
```

Comment:
```suggestion
                test: /\.jsx?$/,
```
получается будут читаться файлы оканчивающиеся на `.j` и `.js`, а надо `.js` и `.jsx` :)